src/Microsoft.AspNet.Mvc.Core/SessionStateTempDataProvider.cs

```diff
@@ -16,8 +17,8 @@ namespace Microsoft.AspNet.Mvc
     /// </summary>
     public class SessionStateTempDataProvider : ITempDataProvider
     {
-        private static JsonSerializer jsonSerializer = new JsonSerializer();
-        private static string TempDataSessionStateKey = "__ControllerTempData";
+        private string TempDataSessionStateKey = "__ControllerTempData";
```

Comment:
I'm partial to the `JsonSerializer` being instance-level, but keys don't benefit from that. In fact this should probably be a `const` since it's private.